test/validators.js

```diff
@@ -8025,4 +8025,67 @@ describe('Validators', () => {
       ],
     });
   });
+
+  it('should validate date', () => {
+    test({
+      validator: 'isDate',
+      valid: [
+        new Date(),
+        new Date([2014, 2, 15]),
```

Comment:
This means we allow users to send created dates using the `new Date()` method? 
I thought, we just need to pass a string. But, it is a good idea.